site/content/en/docs/manual/advanced/consensus.md

```diff
@@ -0,0 +1,173 @@
+      avoid single expert bias in GT, and this is one of the key benefits of the consensus annotation.
+   4. This method requires extra work, but it's much lower than annotating all the dataset several times.
+   5. For instance, annotating 3% of images 5 times will result in extra 3% * (5 - 1) = 12% overhead, comparing to
+      single GT annotations.
+2. In cases of points and poly-line / poly-shape annotations, annotators can add more annotations without a fixed
+   structure, which can introduce significant variation. Similarly, for skeleton annotations, while there is a
+   structure, annotators can move the points around. Due to the presence of numerous points, this can lead to
+   significant variations in annotations for the same data. Using a consensus-based approach for such data types can
+   help reduce these variations, which may arise from minor errors in placing the points.  For example, in the figure,
+   an annotator might place a point representing the foot anywhere across the pink region, leading to significant
+   variation in the annotations. Using such annotations for model training of for model validation is likely to result
+   in reduced performance scores, exactly because of these variations in the correct annotations of the same objects.
+   The consensus annotation approach allows you to minimize such kind of errors by using averaged annotations instead.
+
+![pink_shoes](/images/pink_shoes.jpeg)
+
+
+## Terminology:
+1. `min_iou_threshold`: This is the same as `iou_threshold` param in Quality Settings. It's used for distinction
+   between matched / unmatched annotations. An annotation is considered in the consensus only with the annotations it
+   has an overlap greater than or equal to this
+2. `agreement_score_threshold`: After consensus has produced a merged annotation, it is assigned a score based on it's
+   overlap with other annotations in the cluster, (add label score thing also). Merged annotations with this score
+   lesser than `agreement_score_threshold` are discarded.
+3. `Quorum`: The minimum number of annotations in a consensus for the merging to occur. While deciding the final label
+   of a merged annotation, the count of specific label shouldn't be less than this. Clusters having less than this
+   number of annotations are discarded.
+4. `oks_sigma`: This is the same as `oks_sigma` param in Quality Settings. Like `min_iou_threshold`, but for points.
+   The percent of the bbox area, used as the radius of the circle around the GT point, where the checked point is
+   expected to be. Read more: [https://cocodataset.org/#keypoints-eval](https://cocodataset.org/#keypoints-eval)
+5. `line_thickness`: This is the same as `line_thickness` param in Quality Settings. Thickness of polylines, relatively
+   to the (image area) ^ 0.5. The distance to the boundary around the GT line, inside which the checked line points
+   should be.
+
+
+## Working
```

Comment:
A bit strange name for the section. It describes the algorithm details. Maybe `Algorithm Description`, `Workflow` or `Feature Details` would be better? What do you think?

Reply:
So true, I didn't notice much. `Algorithm Description` sounds good, thanks ^^